### DISCUSSION TITLE
Enable OKTA SSO for kafka-ui

### DISCUSSION THREAD

**Discussion by nareshnelluri** [provectus/kafka-ui#2490] [Q&A] [answered]
Hi All,
Thanks in Advance!!

I am looking for help to configure OKTA authentication for Kafka-ui. I was unable to find the required environment vars to configure  OKTA SSO in docker image properties https://github.com/provectus/kafka-ui#-environment-variables . Found instructions for AUTH0 in the support guide. SPRING_SECURITY_OAUTH2_CLIENT_PROVIDER_AUTH0_AUTHORIZATION_URI
SPRING_SECURITY_OAUTH2_CLIENT_PROVIDER_AUTH0_TOKEN_URI
SPRING_SECURITY_OAUTH2_CLIENT_PROVIDER_AUTH0_USER_INFO_URI
SPRING_SECURITY_OAUTH2_CLIENT_PROVIDER_AUTH0_JWK_SET_URI.

Please provide any instructions documented for OKTA SSO enablement.

**Comment by Haarolean** [comment, marked as answer]
Hi,

there's no special treatment for okta. Just replace "oauth2" with "okta" in these variables.
